frontend/packages/core/src/Table/metadata-table.tsx

```diff
@@ -75,7 +76,7 @@ const TableCell = styled(MuiTableCell)({
   fontSize: "14px",
   fontWeight: "normal",
   height: "48px",
-  padding: "0 16px",
+  padding: "8px 16px",
```

Comment:
added a bit of top and bottom padding, otherwise the the label the row looks like this:

<img width="770" alt="Screen Shot 2022-10-25 at 10 52 40 AM" src="https://user-images.githubusercontent.com/39421794/197807598-3bef175f-a26e-479f-9da7-1826a24cfa87.png">